tracker-radar-tests/TR-domain-matching/domain_matching_tests.json

```
@@ -232,7 +232,8 @@
                 "requestURL": "https://surrogates.test/tracker?abc=2",
                 "requestType": "script",
                 "expectAction": "redirect",
-                "expectRedirect": "data:application/javascript;base64,KGZ1bmN0aW9uKCkge3ZhciB0cmFja2VyPXRydWV9KSgpOw=="
+                "expectRedirect": "data:application/javascript;base64,KGZ1bmN0aW9uKCkge3dpbmRvdy5zdXJyb2dhdGUxPXRydWV9KSgpOw==",
+                "expectExpression": "surrogate1 === true"
```

nit, but for consistency I'd do 

```suggestion
                "expectExpression": "window.surrogate1 === true"
```